lib/gollum-lib/page.rb

```diff
@@ -10,18 +10,23 @@ class << self
       #
       # query     - The String path to match.
       # entry     - The BlobEntry to check against.
-      # global_match - If true, find a File matching path's filename, but not its directory (so anywhere in the repo)
-      def path_match(query, entry, global_match = false, hyphened_tags = false, case_insensitive = false)
+      def global_path_match(query, entry, hyphened_tags = false, case_insensitive = false)
```

Comment:
This method can be used to implement global lookup (see https://github.com/gollum/gollum-lib/pull/437#r1069787205).